### PR TITLE
Cdr 1697 plus sign bug

### PR DESCRIPTION
**Pull request by harringj**

### CODE REVIEW

File: access-common/src/main/java/edu/unc/lib/dl/ui/service/SolrQueryLayerService.java [hunk 1]
```diff
@@ -515,7 +516,8 @@ public void getChildrenCounts(List<BriefObjectMetadata> resultList, AccessGroupS
 		solrQuery.add("f." + ancestorPathField + ".facet.limit", Integer.toString(Integer.MAX_VALUE));
 		// Sort by value rather than count so that earlier tiers will come first in case the result gets cut off
 		solrQuery.setFacetSort("index");
-
+		LOG.warn("This is the one you're looking for!" + solrQuery.toString());
```
Comment (bbpennel) [inline access-common/src/main/java/edu/unc/lib/dl/ui/service/SolrQueryLayerService.java]: Remove

File: access/src/main/resources/log4j.properties [hunk 2]
```diff
@@ -22,6 +22,7 @@ log4j.rootLogger=WARN, SERVER, CDRACCESS
 log4j.logger.org.apache.commons.httpclient=ERROR
 log4j.logger.httpclient=ERROR
 log4j.logger.org.springframework=ERROR
+log4j.logger.edu.unc.lib.dl.search.solr=DEBUG
```
Comment (bbpennel) [inline access/src/main/resources/log4j.properties]: Remove

File: access-common/src/main/java/edu/unc/lib/dl/ui/service/SolrQueryLayerService.java [hunk 3]
```diff
@@ -515,7 +516,8 @@ public void getChildrenCounts(List<BriefObjectMetadata> resultList, AccessGroupS
...
+
```
Comment (harringj) [inline access-common/src/main/java/edu/unc/lib/dl/ui/service/SolrQueryLayerService.java]: I can remove this logging statement, unless you think we should leave something like this in there

File: access/src/main/resources/log4j.properties (hunk 2 above)
Comment (harringj) [inline access/src/main/resources/log4j.properties]: Do we want to leave this change in here?

File: access-common/src/main/java/edu/unc/lib/dl/ui/service/SolrQueryLayerService.java [hunk 4]
```diff
@@ -710,7 +712,7 @@ public HierarchicalBrowseResultResponse getHierarchicalBrowseResults(Hierarchica
 					this.generateSearch(filteredChildrenRequest));
 
 			this.getChildrenCounts(results.getResultList(), accessGroups, "containers",
-					"contentModel:" + SolrSettings.sanitize(ContentModelHelper.Model.CONTAINER.toString()),
+					"resourceType:" + ("(" + ResourceType.Collection.toString() + " OR " + ResourceType.Folder.toString() + ")"),
```
Comment (bbpennel) [inline access-common/src/main/java/edu/unc/lib/dl/ui/service/SolrQueryLayerService.java]: I find the ('s wrapping the "("'s kind of confusing.  Its totally cosmetic and up to you, but removing the outter ('s shouldn't cause any problems.

**Pull request merged**